packages/Popover/src/components/Content/Content.js

```diff
@@ -102,7 +112,7 @@ const Content = React.forwardRef((props, ref) => {
       onMouseOut={handleMouseEvent}
       onMouseOver={handleMouseEvent}
       style={contentStyles}
-      tabIndex="-1"
+      tabIndex="0"
```

Comment:
Required this change in order to receive keyboard events

Reply:
<ContentStyled /> is `Popover.Content` 🤔 how this will mess with the tabbing when the popover is used with the ListBox DatePicker and all those components that depend on this to attach their onKeyDown events?

have you test the keyboard page for the list box and see if this is affecting it?

Reply:
also, you are attaching a new `onKeyUp={handleKeyUp}` but will be overridden by the moreProps, which means this will only work if we don't compose this component with `onKey` events.

Reply:
https://github.com/acl-services/paprika/blob/948aad9abcd0d50ee36edd8d48148878a0292547/packages/ListBox/src/components/Content/Content.js#L67-L73

Reply:
I quickly tested http://storybooks.highbond-s3.com/paprika/uxd-980-popover-esc-key-close-popover-only/?path=/story/forms-listbox-backyard-keyboardtesting--keyboard and seems all is still working but don't know if this implementation of the `esc` just closing the popover will work with the listbox maybe would worth adding an example?

Reply:
Yeah, this will change the keyboard use patterns, and I don't think it's needed.  Seems to still work with `tabIndex="-1"`

Reply:
Nahum' has a good point about `moreProps` overriding handlers, etc on the Popover.Content.  This might be a good opportunity to move `moreProps` into a more deliberate order so that can't happen.  

However, it's also true that we might _want_ to add custom handlers, in which case we should compose them with the ones we define to make sure the Popover doesn't break.  There's a `callAll()` function in `@paprika/helpers`, like the one from the Epic React course, that could help with this.

Reply:
So i'm seeing that the down/up arrow isn't preselecting the listbox options in /forms-listbox-backyard-keyboardtesting--keyboard story so this will need some more understanding of what list box is expecting.

Reply:
☝️  This issue should be fixed now